### ISSUE TITLE
Installs as "wrong" user

### ISSUE THREAD

**Issue by jpambrun** [rancher-sandbox/rancher-desktop#613]
I am part of a big organization and my computer is heavily managed by IT. I do have admin privileges, but on a second account that I cannot log into (i.e. I can only use it through UAC to install software). Specifically, I have a normal **username** and unreachable good-only-for-uac **username_admin** account.

The issue is that while logged in as **username** when installer prompts for UAC, I have to enter my **username_admin** and password credential. It then happily installs, but unfortunately it installs under C:\Users\\**unsername_admin**\\AppData\Local\Programs\Rancher Desktop and is unreachable to my normal user. 

Is there any way to either 1) install for all users or 2) make sure that only the required steps are performed as admin and that it installs under the currently logged in user?

**Comment by mattfarina** [comment]
This is an issue. We should handle this cleanly. I've added it to our agenda to address.

**Comment by jpambrun** [comment]
For what it's worth, I could copy the files from C:\Users\unsername_admin\AppData\Local\Programs\Rancher Desktop to my normal user and it seems to run okay from there:

```
kubectl get pods --all-namespaces
NAMESPACE     NAME                                      READY   STATUS      RESTARTS   AGE
kube-system   helm-install-traefik-crd-vbd77            0/1     Completed   0          71s
kube-system   helm-install-traefik-h8cg6                0/1     Completed   1          71s
kube-system   svclb-traefik-2nsns                       2/2     Running     0          65s
kube-image    builder-r75g2                             2/2     Running     0          70s
kube-system   coredns-7448499f4d-z8xf2                  1/1     Running     0          70s
kube-system   local-path-provisioner-5ff76fc89d-vf7lg   1/1     Running     0          70s
kube-system   metrics-server-86cbb8457f-rzmjb           1/1     Running     0          70s
kube-system   traefik-97b44b794-jj2td                   1/1     Running     0          65s
```

But I already had kubectl and more installed and in path.

**Comment by jpambrun** [comment]
If anyone face the same issue, I had to add

```
C:\Users\username\AppData\Local\Programs\Rancher Desktop\resources\resources\win32\bin
```

to the my path to get kim working

**Comment by mook-as** [comment]
We could install things system wide (`C:\Program Files`), but that would make upgrades messier (since users wouldn't be able to write to the install directory.)

I think ideally we'd need to change the installer to not require elevation, and only request elevation if we detect that we need to install WSL-related components. (Spawning a helper at that point, of course.)

**Comment by mook-as** [comment]
Fixed by #632.
